src/app/(proper_react)/(redesign)/(authenticated)/user/(dashboard)/settings/actions.ts

```diff
@@ -138,7 +139,7 @@ export async function onRemoveEmail(email: EmailRow) {
     return {
       success: false,
       error: "delete-email-without-active-session",
-      errorMessage: `Tried to delete email [${email.id}] without an active session.`,
+      errorMessage: `User tried to delete email without an active session.`,
```

Comment:
I guess related to above pondering&hellip; Should this string be in an FTL file similar to L167 below?